lib/plugins/entities.js

```diff
@@ -919,7 +941,14 @@ function inject (bot) {
   }
 
   function fetchEntity (id) {
```

Comment:
I think entity handling could be completely refactored, fetchEntity may not be needed if the relevant entity API storage is in p-world